addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: Fisher
+ * Toggle Simulation on object (runs on server only via module framework).
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params ["_logic"];
+
+private _object = attachedTo _logic;
+if (isNull _object) then {
+    [LSTRING(NoObjectSelected)] call EFUNC(common,displayTextStructured);
```

Comment:
Also a good catch, hmm, I suppose it probably should run on the client and then send an event to the server so that this client feedback can be given.